setup.py

```diff
@@ -11,4 +11,5 @@
   packages=[
         'async_mock'
     ],
+  python_requires='>=3.10',
```

Comment:
Let's set this to 3.8 so clients can upgrade to this version without upgrading to python 3.10.